src/containers/AccountConnection.jsx

```diff
@@ -220,7 +216,10 @@ class AccountConnection extends Component {
 
   render () {
     const { t, existingAccount, connector, fields } = this.props
-    const { submitting, deleting, error, credentialsError, isSuccessTimedOut } = this.state
+    const { submitting, deleting, error, isSuccessTimedOut } = this.state
+    const hasGlobalError = error && error.message && ![
+      ACCOUNT_ERRORS.LOGIN_FAILED
+    ].includes(error.message)
```

Comment:
It would, but it's about a generic error so I wanted to keep it compliant with later specific errors.